reagurk/public/index.html

```diff
@@ -4,7 +4,22 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+
+      <meta property="og:type" content="website">
+      <meta property="og:url" content="https://gurkult.com">
+      <meta property="og:title" content="Gurkult">
+      <meta property="og:site_name" content="Gurkult" />
+      <meta property="og:locale" content="en_US" />
+      <meta property="og:description" content="The Gurkult is an open source community website.">
+      <meta property="og:image" content="https://gurkult.com/logo.png">
+      <meta propert="og:image:alt" content ="Gurkult Logo">
+      <meta name="twitter:card" content="summary_small_image"/>
+      <meta property="twitter:image" content="https://gurkult.com/logo.png">
+      <meta property="twitter:title" content="Gurkult">
+      <meta property="twitter:image:alt" content="Gurkult Logo">
+      <meta name="theme-color" content="#10B981" />
+      
+
     <meta
```

Comment:
Why is there a gap above this? And why is this separated from the rest of the meta tags?